website/docs/docs/build/metricflow-time-spine.md

```diff
@@ -13,13 +13,13 @@ MetricFlow requires you to define a time-spine table as a model-level configurat
 - [Metric offsets](/docs/build/derived#derived-metric-offset)
 - [Conversion metrics](/docs/build/conversion)
 - [Slowly Changing Dimensions](/docs/build/dimensions#scd-type-ii)
-- [Metrics](/docs/build/metrics-overview) with the `join_to_timespine` configuration set to true
-To see the generated SQL for the metric and dimension types that use time-spine joins, refer to the respective documentation or add the `compile=True` flag when querying the Semantic Layer to return the compiled SQL.
+- [Metrics](/docs/build/metrics-overview) with the `join_to_timespine` configuration set to `true`. To see the generated SQL for the metric and dimension types that use time-spine joins, refer to the respective documentation or add the `compile=True` flag when querying the Semantic Layer to return the compiled SQL.
 
-#### Configuring time-spine
+### Configuring time-spine
```

Comment:
Is there a reason we use "time-spine" instead of just "time spine"? Curious if @Jstein77 has an opinion on this